Marlin/src/core/serial.h

```diff
@@ -37,6 +37,7 @@ extern const char NUL_STR[],
                   SP_I_LBL[], SP_J_LBL[], SP_K_LBL[],
                   SP_P_STR[], SP_T_STR[],
                   X_STR[], Y_STR[], Z_STR[], E_STR[],
+                  I_STR[], J_STR[], K_STR[],
```

Comment:
These are commonly-used `PROGMEM` strings defined in `serial.cpp` which are meant to be used with the `SERIAL_ECHOPAIR_P` style macros that take `PROGMEM` strings instead of bare strings that are then wrapped in `PSTR()` as with `SERIAL_ECHOPAIR`. These exist due to paranoia about the way the compiler manages string storage, to prevent many duplicates of the same string.